editor/assets/primitives.fbx.meta

```diff
@@ -74,7 +74,7 @@
       "subMetas": {},
       "userData": {
         "gltfIndex": 2,
-        "triangleCount": 72
+        "triangleCount": 92
```

Comment:
Why change it?

Reply:
I don't know. It's the editor changed the meta.

Reply:
I will revert it.

Reply:
Reverted.